node/test/PackageSourceTests.ts

```diff
@@ -17,6 +17,40 @@ const testSchemaB = "org.opent2t.test.schemas.b";
 const testTranslatorOne = "org.opent2t.test.translators.one";
 const testTranslatorTwo = "org.opent2t.test.translators.two";
 
+const testOnboardingPackageOne = "opent2t-onboarding-org-opent2t-test-onboarding-one";
+const testOnboardingOnboardingOne = "org.opent2t.test.onboarding.one";
+const testOnboardingSchemaOne = "org.opent2t.test.onboarding.one";
+
+
+test("Get onboarding One info", async t => {
+    let packageInfo: PackageInfo | null = await testPackageSource.getPackageInfoAsync(testOnboardingPackageOne);
```

Comment:
@jasongin When LocalPackageSource is supposed to work on 'git directory structure' , why does it take npm module name of it ? like, opent2t-onboarding-org-opent2t-test-onboarding-one
Could you please layout the exact use case here ?